CHANGELOG.md

```diff
@@ -211,7 +211,7 @@
 
 - Better support for rebasing branches - we now always treat the last build on this branch as a baseline, even if strictly it is not a git ancestor of the current commit. This helps deal with the situation where you rebase a branch off master, and still want to use the previously approved snapshots.
 
-- Improved support for CI systems, especially _Netlify_ and _Travis PR builds_. Travis PR builds are a special case, read more about how to handle them in Chromatic here: https://docs.chromaticqa.com/setup_ci#travis
+- Improved support for CI systems, especially _Netlify_ and _Travis PR builds_. Travis PR builds are a special case, read more about how to handle them in Chromatic here: https://docs.chromatic.com/setup_ci#travis
```

Comment:
@ndelangen at least the `/setup_ci` path has changed (it's now just `/ci`). Please check the urls you link to against the latest docs here: https://github.com/chromaui/chromatic-docs/tree/edits . The names of the md files just map to /name

Reply:
@zol are you going to make the old URLs redirect?